src/Components/Blazor/testassets/Wasm.Authentication.Server/Data/Migrations/20200123141439_Initial.cs

```diff
@@ -0,0 +1,278 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace Wasm.Authentication.Server.Data.Migrations
+{
+    public partial class Initial : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
```

Comment:
Presumably all this autogenerated migration stuff is to enable E2E testing that includes identity, is it?

If there was some way of not having the migrations here that would be great for future maintainability, but I appreciate this might be the only practical option. Do you have any ideas on how we could achieve eliminating this from here?